storage/federatedx/ha_federatedx.cc

```diff
@@ -1489,34 +1489,33 @@ static void fill_server(MEM_ROOT *mem_root, FEDERATEDX_SERVER *server,
        sizeof(int) + 8);
   key.append(scheme);
   key.q_append('\0');
-  server->hostname= (const char *) (intptr) key.length();
+  size_t hostname_sz= key.length();
```

Comment:
I don't care much and it is up to you.

But this is not "hostname size", it's "size of all previous parts of the key" or "hostname start position". So, `hostname_offset` or `hostname_start` or `hostname_pos`.

Same below.